api/go.mod

```diff
@@ -1,16 +1,15 @@
 module github.com/aquasecurity/tracee/api
 
-go 1.21
+go 1.22.3
```

Comment:
Maybe a silly question, but why do we need to specify the patch version? won't `1.22` be enough?

Reply:
I decided to use toolchain only in the main go.mod. The others when imported by tracee will have already the toolchain version enforced.